app/Http/Controllers/User/BlogController.php

```diff
@@ -5,14 +5,16 @@
 use App\Http\Controllers\Controller;
 use App\Blog;
 use App\User;
+use Exception;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
 
 class BlogController extends Controller
 {
 
     // return information user
-    public function defineUser() {
+    public function defineUser()
+    {
```

Comment:
why have this format changing ?

Reply:
Dạ em check phpcs thi bị lỗi, em viết lại theo format PSR2.

Reply:
trả lại như cũ (vì ko phải code sửa trong lần này)

Reply:
I fixed